customcommands/customcommands.go

```diff
@@ -160,11 +160,15 @@ func (cc *CustomCommand) Validate(tmpl web.TemplateData) (ok bool) {
 		return false
 	}
 
-	if cc.TriggerTypeForm == "interval_minutes" && cc.TimeTriggerInterval < 5 {
-		tmpl.AddAlerts(web.ErrorAlert("Minimum interval is now 5 minutes (was recently from 1)"))
+	if cc.TriggerTypeForm == "interval_minutes" && (cc.TimeTriggerInterval < 5 || cc.TimeTriggerInterval > 44640) {
+		tmpl.AddAlerts(web.ErrorAlert("Minute intervals can be between 5 and 44640(1 Month)"))
```

Comment:
Perhaps
```go
const MaxIntervalDur = int(time.Hour * 24 * 31)
```
...and for minutes `MaxIntervalDur/int(time.Minute)`; similar for hours.

Reply:
```
// Setting it to 1 Month approx
const (
	MinIntervalTriggerDurationMinutes = 5
	MinIntervalTriggerDurationHours   = 1
	MaxIntervalTriggerDurationHours   = 744
	MaxIntervalTriggerDurationMinutes = 44640
)
```

Just doing this